### ISSUE TITLE
Gather Peer statistics per torrent

### ISSUE THREAD

**Issue by vijayanandnandam** [anacrolix/torrent#717]
Is there any way to gather peer stats like seeds, leechers using the tracker scraper API using udp / http / https ?
I am unable to find any examples..
Something similar to this https://github.com/medariox/scrapeer

**Comment by anacrolix** [comment]
Yes, scraping has only been implemented for UDP at present (https://pkg.go.dev/github.com/anacrolix/torrent@v1.40.0/tracker/udp#Client.Scrape), and isn't included in the transport-agnostic interface in `./tracker`. The docs are sparse, but there should be some examples of its usage in `./cmd/...`. PRs, including for docs are welcome.

**Comment by anacrolix** [comment]
There may not be any publicly available examples of its usage.
